### ISSUE TITLE
Toonily.com chapter no longer loading

### ISSUE THREAD

**Issue by kitsame** [tachiyomiorg/tachiyomi-extensions#15889]
### Source information and language

Toonily 1.4.37

### Steps to reproduce

As the title said, chapters on toonily.com are no longer loading, All it says is "The image couldn't be loaded" with Retry button. Download is also not working. Webview is working fine without any problem so is on computer.
Steps to reproduce
1. Go to Solo Leveling or any manhwa there
2. Choose any chapter you want

Thought that I was blocked by cloudflare or something similar but webview works fine. Did the cloudlfare troubleshooting just in case, didn't work
Updating chapters works normally which wouldn't work if I was blocked by the site.

### Expected behavior

To see images

### Actual behavior

You don't see any images, only "The image couldn't be loaded" with Retry button

### Tachiyomi version

0.14.5

### Android version

Android 11

### Other details

![Screenshot_2023-03-30-11-51-24-735_eu kanade tachiyomi](https://user-images.githubusercontent.com/4948872/228799206-c096e3f4-3eb6-42fe-ba94-303c3603b924.jpg)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.14.5](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by kitsame** [comment]
Like I said, i THOUGH it WAS cloudflare, and tried cloudflare solutions, it was not cloudflare...
